styleguide/_themes/derek/scss/snowflakes/_homepage.scss

```diff
@@ -1,65 +1,76 @@
 .homepage-hero {
-  @extend .full-width;
-  height: 100%;
-  overflow: visible;
-  background-repeat: no-repeat;
-  top: 65px;
-  margin-bottom: 40px;
-  background-image: url("https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/backgrounds/homepage.jpg");
+  @extend %full-width;
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/backgrounds/homepage.jpg');
   background-position: 25% 50%;
-  position: relative;
+  background-repeat: no-repeat;
   background-size: auto;
   border-bottom: 0;
+  height: 100%;
+  margin-bottom: 40px;
+  overflow: visible;
+  position: relative;
+  top: 65px;
+
   h2 {
-    padding-top: 20px;
+    @extend %white;
+    @extend %left;
     font-size: 34px;
-    line-height: 40px;
     font-weight: 300;
-    @extend .white;
-    @extend .left;
+    line-height: 40px;
+    padding-top: 20px;
   }
-  p{
+
+  p {
+    font-size: 16px;
     letter-spacing: 1px;
     line-height: 24px;
-    font-size: 16px;
     padding-bottom: 10px;
   }
 }
 
-#hosting-options{
-  text-align:center;
-  .product{
-    p{ margin-bottom:30px; }
-    &:hover{
-      @extend .bg-light-gray;
-      hr{ border-color:$mediumteal; }
-      h3, a{ color:$mediumteal; }
+#hosting-options {
+  text-align: center;
+
+  .product {
+    p { margin-bottom: 30px; }
+
+    &:hover {
+      @extend %bg-light-gray;
+      hr { border-color: $mediumteal; }
+
+      h3,
+      a {
+        color: $mediumteal;
+      }
     }
   }
 }
 
-#solutions{
-  a{ text-decoration:none; }
-  .solution:hover{
-    .border-white{
-     border-color:$mustard;
-     i{ color:$mustard; }
+#solutions {
+  a { text-decoration: none; }
```

Comment:
The homepage scss file is getting a rewrite today so I wouldnt worry about this one right now guys.